integration-tests/src/test/java/tachyon/client/FileOutStreamIntegrationTest.java

```diff
@@ -78,6 +78,9 @@ public final void after() throws Exception {
 
   @Before
   public final void before() throws IOException {
+    // Disable hdfs client caching to avoid file system close() affecting other clients
+    System.setProperty("fs.hdfs.impl.disable.cache", "true");
```

Comment:
I think the right fix might be to keep this in before class but put the clear property in `AfterClass`?